tests/test_minmax.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# ================================================================================================
+
+import unittest
+import dimod
+import numpy as np
+
+num_vars = 1000
+
+class TestMinMax(unittest.TestCase):
```

Comment:
I would probably add these tests next to BQM methods tests in [`test_bqm.py`](https://github.com/dwavesystems/dimod/blob/master/tests/test_bqm.py) or [`test_binary_quadratic_model.py`](https://github.com/dwavesystems/dimod/blob/master/tests/test_binary_quadratic_model.py).

Reply:
`test_bqm.py` please :smile: